admin_guide/upgrading_citus.rst

```diff
@@ -63,14 +63,20 @@ Step 1. Update Citus Package
 Step 2. Apply Update in DB
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-After installing the new package, run the extension upgrade script:
+After installing the new package and restarting the database, run the extension upgrade script.
 
 .. code-block:: bash
 
+  # you must restart PostgreSQL before running this
   psql -c 'ALTER EXTENSION citus UPDATE;'
 
-  psql -c '\dx'
   # you should see the newer Citus version in the list
+  psql -c '\dx'
+
+
+.. note::
+
+  During a major version upgrade, from the moment of yum installing a new version, Citus will refuse to run distributed queries until ALTER EXTENSION is executed. This is to protect your data, as Citus object and function definitions are specific to a version. After a yum install you should (a) restart and (b) run alter extension.
```

Comment:
"Citus will refuse to run distributed queries until ALTER EXTENSION is executed." -> "Citus will refuse to run distributed queries until the *database is restarted* and ALTER EXTENSION is executed."